gef-ui/server/controllers/about-facility/get-next-page-helper.ts

```diff
@@ -0,0 +1,21 @@
+import { isFacilityEndDateEnabledOnGefVersion, parseDealVersion } from '@ukef/dtfs2-common';
+
+type GetNextPageParams = {
+  dealId: string;
+  facilityId: string;
+  dealVersion: number | undefined;
+  isUsingFacilityEndDate: boolean;
+};
+
+export const getNextPage = ({ dealId, facilityId, dealVersion, isUsingFacilityEndDate }: GetNextPageParams): string => {
```

Comment:
Should we add documentation or do we have a a "self documenting" style approach?

Reply:
I think this is self documenting enough -- I tend to always like self documenting whenever possible as it means its most up to date :)